bootstrap.sh

```diff
@@ -114,7 +114,12 @@ create_symlinks() {
     lnif "$source_path/.vimrc.bundles" "$target_path/.vimrc.bundles"
     lnif "$source_path/.vimrc.before"  "$target_path/.vimrc.before"
     lnif "$source_path/.vim"           "$target_path/.vim"
-
+    
+    if has('nvim')
```

Comment:
It seems like your approach (with `has('nvim')` would only install the nvim symlink if we were running nvim, but not if we had nvim installed, but were executing it with vim.

It should check for nvim in the `$PATH` via `type` or `which`. We currently use `type` in `program_exists()`.